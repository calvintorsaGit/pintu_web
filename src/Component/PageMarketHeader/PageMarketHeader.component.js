import {Search} from 'semantic-ui-react'

const PageMarketHeader = () => {
    return <div class='flex flex-row mt-4'>
        <h2 class='flex-1 text-4xl text-left'>Harga Crypto dalam Rupiah Hari Ini</h2>
        <Search placeholder='Cari aset di Pintu...' size={"big"}/>
    </div>
}

export default PageMarketHeader
