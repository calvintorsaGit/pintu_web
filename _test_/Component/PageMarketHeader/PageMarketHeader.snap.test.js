import renderer from 'react-test-renderer';
import React from "react";

import PageMarketHeader from "../../../src/Component/PageMarketHeader/PageMarketHeader.component";

it('renders correctly', () => {
    const tree = renderer
        .create(<PageMarketHeader/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
