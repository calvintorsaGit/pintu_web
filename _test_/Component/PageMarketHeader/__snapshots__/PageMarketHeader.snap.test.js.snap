// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="flex flex-row mt-4"
>
  <h2
    class="flex-1 text-4xl text-left"
  >
    Harga Crypto dalam Rupiah Hari Ini
  </h2>
  <div
    className="ui big search"
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseDown={[Function]}
  >
    <div
      className="ui icon input"
    >
      <input
        autoComplete="off"
        className="prompt"
        onChange={[Function]}
        onClick={[Function]}
        placeholder="Cari aset di Pintu..."
        tabIndex="0"
        type="text"
        value=""
      />
      <i
        aria-hidden="true"
        className="search icon"
        onClick={[Function]}
      />
    </div>
    <div
      className="results transition"
    >
      <div
        className="message empty"
      >
        <div
          className="header"
        >
          No results found.
        </div>
      </div>
    </div>
  </div>
</div>
`;
